superdesk/io/feeding_services/ftp.py

```diff
@@ -79,8 +97,14 @@ def _update(self, provider, update):
             except KeyError:
                 logger.warning('missing move_path')
                 move_dest_path = None
+            try:
+                move_dest_path_error = os.path.join(config.get('path', ''), config['move_path_error'])
```

Comment:
@petrjasek so what do you think? I can remove the ability to disable _move on error_.
If I add default dir, I'll add mkdir if the dir doesn't exist, I think it would make sense in this case.

So do I let as it is or add default dir and dir creation if it doesn't exist? Either way is OK to me.
@karelpetrak your input is welcome here.